src/controllers/authController.js

```diff
@@ -0,0 +1,34 @@
+import models from "../database/models/index.js";
+const {Users} = models;
+import helper from "../utils/jwtHelper.js";
+const { comparePassword } = require("../utils");
+
+    const login = async (req, res, next) =>{
+        try{
+        const { email, password} = req.body;
+        const user = await Users.findOne({where:{email}});
+        if(!user){
+        return res.status(400).send({error: "Email not found"}); 
+        }
+        if(user && comparePassword(password, user.password)){
+            const userInfo = { 
+            id: user.id, 
+            fullname: user.fullname,
+            username: user.username,
+            role: user.role,
+            email:user.email 
+            } 
+        console.log(userInfo);
```

Comment:
I think you forgot to remove this console.log(userInfo) because it is like leaking user data.